test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -23,19 +23,22 @@
 
 - name: SchedulingPodAntiAffinity
   workloadTemplate:
+  - opcode: createNamespaces
```

Comment:
I'm a little curious that if we `createNamespaces` before `createPods` in the `workloadTemlate` strictly like you add here. Do the issues raised in https://github.com/kubernetes/kubernetes/pull/107700 still exist?

Reply:
My first thought was:
If users don't specify `namespace` in `createPods`, scheduler_perf defines a namespace for Pod automatically. ​#107700 changes scheduler_perf to create namespaces that are automatically defined. 
On the other hand, the case users specify `namespace` in `createPods`, it means "scheduler_perf doesn't define a namespace for Pod automatically.". In that cases, I thought users should create namespace by themselves with `createNamespaces`.

---

But, on second thought, it would be more convenient if scheduler_perf creates namespaces even when users specify `namespace` in `createPods`. So, I'll revert these changes and change scheduler_perf to do that.